examples/ping-pong/src/main.rs

```diff
@@ -39,6 +39,8 @@ async fn main() {
     /// the handler itself is being passed into the Gossip Agent, so it needs to be static as well.
     pub static GOSSIP_AGENT: OnceCell<GossipAgent> = OnceCell::new();
 
+    let waku_host = env::var("WAKU_HOST").ok();
```

Comment:
Is there a reason for adding these to the 🏓  Radio? I think it should stay as small and simple as possible to serve as a first step to Radio development

Reply:
Yes, this allows us to test the SDK across host and ports with this example. If not provided then the radio will just use the simple fallback at localhost:60000 so there's no extra complexity introducing these variables